### PR TITLE
fix(gatsby): Code samples in getting started page and wizard

### PR DESCRIPTION
**Pull request by iker-barriocanal**
Fixes the Gatsby code samples in the getting started page and onboarding wizard.

### CODE REVIEW

**Review by kamilogorek — APPROVED** [review]
Good Hacktoberfest PR :P

File: src/platforms/javascript/guides/gatsby/index.mdx [hunk 1]
```diff
@@ -37,8 +37,8 @@ Register the plugin in your Gatsby configuration file (typically `gatsby-config.
       },
     },
     // ...
-  ];
-}
+  ],
+};
```
Comment (rhcarvalho) [inline src/platforms/javascript/guides/gatsby/index.mdx:41]: Hmm maybe we go ahead and make this closer to Gatsby docs -- https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/

```
module.exports = {
  plugins: [
    //plugins go here
  ],
}
```

Comment (AbhiPrasad) [reply src/platforms/javascript/guides/gatsby/index.mdx:41]: I'd rather we stay consistent with our code styling throughout docs.

Comment (rhcarvalho) [reply src/platforms/javascript/guides/gatsby/index.mdx:41]: We declare this file to be `gatsby-config.js`. Does it actually work with Gatsby without the `module.exports` part? For the rest -- yeah I think we should have a `prettier` config in this repo so just apply that for consistency is okay.

Comment (AbhiPrasad) [reply src/platforms/javascript/guides/gatsby/index.mdx:41]: My mistake, I misunderstood the nature of your comment. Yeah we need to add `module.exports` as it's a javascript file.

File: src/wizard/javascript/gatsby.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ npm install --save @sentry/gatsby
 Register the plugin in your Gatsby configuration file (typically `gatsby-config.js`).
 
 ```javascript
-{
+module.exports = {
```
Comment (rhcarvalho) [inline src/wizard/javascript/gatsby.md:23]: Could you add the file name here to?

**Review by rhcarvalho — APPROVED** [review]
Thanks!

File: src/platforms/javascript/guides/gatsby/index.mdx (hunk 1 above)
Comment (iker-barriocanal) [reply src/platforms/javascript/guides/gatsby/index.mdx:41]: I also missed the `module.exports` hehe. Should be fixed now, including the onboarding wizard. LMK what you think.

File: src/wizard/javascript/gatsby.md (hunk 2 above)
Comment (rhcarvalho) [reply src/wizard/javascript/gatsby.md:23]: IIRC we have other cases like that. Adding the filename changes how the code snippet renders, see even https://github.com/getsentry/sentry-docs/pull/4232/files#diff-eaab05775310590a0a9a65d2fe287113f6c72777be1b2c7af70c84b162a8a26aR27 is like that.

Comment (iker-barriocanal) [reply src/wizard/javascript/gatsby.md:23]: Alright! Updated.

**Pull request merged**